src/components/storage/InstallationDestination.jsx

```diff
@@ -433,7 +436,13 @@ export const InstallationDestination = ({
                                 : _("No usable disks detected")
                         )}
                     {rescanDisksButton}
-                    {!isBootIso && <ModifyStorage idPrefix={idPrefix} onCritFail={onCritFail} onRescan={onClickRescan} />}
+                    <ModifyStorage
+                      idPrefix={idPrefix}
+                      onCritFail={onCritFail} onRescan={onClickRescan}
```

Comment:
Just a little nitpick, onRescan could go to a separate line.

Reply:
Fixed :)